acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/slashing/SlashingImportAcceptanceTest.java

```diff
@@ -0,0 +1,123 @@
+import tech.pegasys.web3signer.core.signing.KeyType;
+import tech.pegasys.web3signer.dsl.signer.Signer;
+import tech.pegasys.web3signer.dsl.signer.SignerConfigurationBuilder;
+import tech.pegasys.web3signer.dsl.utils.MetadataFileHelpers;
+import tech.pegasys.web3signer.tests.AcceptanceTestBase;
+
+import java.io.File;
+import java.math.BigDecimal;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.io.Resources;
+import org.apache.tuweni.bytes.Bytes;
+import org.jdbi.v3.core.Jdbi;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+public class SlashingImportAcceptanceTest extends AcceptanceTestBase {
+
+  private static final MetadataFileHelpers metadataFileHelpers = new MetadataFileHelpers();
+  public static final String DB_USERNAME = "postgres";
+  public static final String DB_PASSWORD = "postgres";
+  public static final String VALIDATOR_PUBLIC_KEY =
+      "0x8f3f44b74d316c3293cced0c48c72e021ef8d145d136f2908931090e7181c3b777498128a348d07b0b9cd3921b5ca537";
+  public static final String ATTESTATION_SIGNING_ROOT =
+      "0x30752da173420e64a66f6ca6b97c55a96390a3158a755ecd277812488bb84e57";
+  public static final String BLOCK_SIGNING_ROOT =
+      "0x4ff6f743a43f3b4f95350831aeaf0a122a1a392922c45d804280284a69eb850b";
+
+  protected final BLSKeyPair keyPair = BLSKeyPair.random(0);
+
+  void setupSigner(final Path testDirectory, final boolean enableSlashing) {
+    final SignerConfigurationBuilder builder =
+        new SignerConfigurationBuilder()
+            .withMode("eth2")
+            .withSlashingEnabled(enableSlashing)
+            .withSlashingProtectionDbUsername(DB_USERNAME)
+            .withSlashingProtectionDbPassword(DB_PASSWORD)
+            .withMetricsEnabled(true)
+            .withKeyStoreDirectory(testDirectory);
+
+    final Path keyConfigFile = testDirectory.resolve("keyfile.yaml");
+    metadataFileHelpers.createUnencryptedYamlFileAt(
+        keyConfigFile, keyPair.getSecretKey().toBytes().toHexString(), KeyType.BLS);
+
+    startSigner(builder.build());
+  }
+
+  @Test
+  void slashingDataIsExported(@TempDir Path testDirectory) throws URISyntaxException {
+    setupSigner(testDirectory, true);
+
+    final Path importFile =
+        new File(Resources.getResource("slashing/slashingImport.json").toURI()).toPath();
+
+    final SignerConfigurationBuilder builder = new SignerConfigurationBuilder();
+    builder.withMode("eth2");
+    builder.withSlashingEnabled(true);
+    builder.withSlashingProtectionDbUrl(signer.getSlashingDbUrl());
+    builder.withSlashingProtectionDbUsername("postgres");
+    builder.withSlashingProtectionDbPassword("postgres");
+    builder.withKeyStoreDirectory(testDirectory);
+    builder.withSlashingImportPath(importFile);
+    builder.withHttpPort(12345); // prevent wait for Ports file in AT
+
+    final Signer importSigner = new Signer(builder.build(), null);
```

Comment:
why not map the file into the InterchangeFormat struct we have in a test fixture - and compare db against that?

means we don't need to compare directly against hard-coded expected values in the json file.

Reply:
done